vllm/platforms/interface.py

```diff
@@ -12,6 +12,17 @@ class PlatformEnum(enum.Enum):
     UNSPECIFIED = enum.auto()
 
 
+class DeviceCapability(NamedTuple):
+    major: int
+    minor: int
+
+    def as_version_str(self) -> str:
+        return f"{self.major}.{self.minor}"
+
+    def __int__(self) -> int:
+        return self.major * 10 + self.minor
```

Comment:
@DarkLight1337 what's this? an `__init__` method returns an `int`?

Reply:
It's the `__int__` dunder method that is used when `int(...)` is called on the object.

Reply:
oh, it is `__int__` , not `__init__` , very confusing :(

Reply:
can we avoid this?

Reply:
See if [61710aa](https://github.com/vllm-project/vllm/pull/8534/commits/61710aa2806b043b515c69c327ec64ebca58b070) clears it up for you.

Reply:
not really, I think we should not convert this object to int.

Reply:
Would using `to_int()` directly work for you?

Reply:
if any file uses `int(DeviceCapability)`, we should change that usage.

Reply:
Updated.